tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.md

```diff
@@ -100,12 +104,13 @@ cat: can't open 'mydata/test_nfs_file.txt': No such file or directory
 ```
 Error response from daemon: volume ${nfsNamedVolume} in use by
 ```
-* Steps 34 - 36 should result in success; step 36 should kill/drop the server
-* Step 37 should result in error with the following message:
+* Steps 34-38 should result in success
+* Steps 39 - 41 should result in success; step 41 should kill/drop the server
+* Step 42 should result in error with the following message:
 ```
 Server error from portlayer: unable to wait for process launch status:
```

Comment:
We do not want to expect that error. We want an informative error, write it with that in mind and file a bug for something that helps the user out. And create a github issue check in the test for this message when it gets fixed.